apps/dashboard/app/(app)/projects/[projectId]/deployments/components/table/components/env-status-badge.tsx

```diff
@@ -10,11 +10,12 @@ const statusBadgeVariants = cva(
       variant: {
         enabled: "text-successA-11 bg-successA-3",
         disabled: "text-warningA-11 bg-warningA-3",
-        current: "text-feature-11 bg-feature-4",
+        live: "text-feature-11 bg-feature-4",
+        rolledBack: "text-warningA-11 bg-warningA-2",
```

Comment:
`bg-warningA-3` might be a better option. I couldn't even tell if this had a background with this `bg-warningA-2`

bg-2 screenshot
<img width="404" height="113" alt="image" src="https://github.com/user-attachments/assets/e00013a1-7383-4aa3-9529-8b9fdc4f29ac" />